analyzers/src/SonarAnalyzer.CSharp/Rules/PropertyToAutoProperty.cs

```diff
@@ -185,8 +174,8 @@ private static bool TryGetFieldFromGetter(AccessorDeclarationSyntax getter, Sema
                 semanticModel, out getterField);
 
             ExpressionSyntax GetReturnExpressionFromBody(BlockSyntax body) =>
-                body?.Statements.Count == 1 &&
-                body.Statements[0] is ReturnStatementSyntax returnStatement
+                body?.Statements.Count == 1
```

Comment:
_Line 182 below:_ Let's just remove the unneeded indirection. It makes the simple `?.Expression` it really hard to read.

Reply:
This wasn't addressed.